Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -254,6 +260,11 @@ def set_test_data(self, data):
             self.error(1, "Test data input requires a class variable")
             data = None
 
+        self.warning(5)
+        if data and np.isnan(data.Y).any():
+            self.warning(5, "Test data has missing values in class variable(s)")
```

Comment:
Instances with unknown target values were removed from test data.

Also, consider joining the two warnings if it's not too complicated.